.editorconfig

```diff
@@ -11,6 +11,7 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+spelling_exclusion_path = spelling.dic
```

Comment:
Could the file be named `exclusion.dic`? I believe that's the standard filename based on the documentation here: https://learn.microsoft.com/en-us/visualstudio/ide/text-spell-checker?view=vs-2022#how-to-customize-the-spell-checker